src/main/java/uk/gov/hmcts/reform/data/ingestion/camel/service/AuditServiceImpl.java

```diff
@@ -55,24 +61,30 @@
      */
     public void auditSchedulerStatus(final CamelContext camelContext) {
 
+        List<String> nonStaleFiles = archivalFileNames.stream().filter(file ->
+            getFileName(camelContext, file) != null && getFileName(camelContext, file).equals(file))
```

Comment:
getFileName(camelContext, file) != null is already verified inside the method so not required it seems here?

Reply:
I explicitly returned Null from getFileName.Let me update and return Empty from there. and will update this.